src/disco/css/InstallButton.scss

```diff
@@ -190,3 +170,19 @@ $installStripeColor2: #00C42E;
     }
   }
 }
+
+.progress {
+  background: transparentize(#3ef827, 0.50);
+  bottom: 0;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 0;
+}
+
+@for $i from 1 through 100 {
+  [data-download-progress="#{$i}"] .progress {
+    width: #{percentage($i) / 100};
```

Comment:
I tried manually driving the download progress using hot reloading and it doesn't have a transition on it. We might want this to be a `translateX(percentage($i - 100))` and always be 100% wide (and add a transition so it's less jumpy).